pyproject.toml

```diff
@@ -84,3 +86,78 @@ local_scheme = 'no-local-version'
 addopts = "--cov=plenoptic -n auto"
 testpaths = ["tests"]
 
+[tool.ruff]
+extend-include = ["*.ipynb"]
+src = ["src", "tests"]
+# Exclude a variety of commonly ignored directories.
+exclude = [
```

Comment:
I think most of these can be removed. maybe except `.git` and `.github`, but ruff automatically ignores whatever's in `.gitignore`. for those here that are relevant (e.g., not `node_modules`), they belong in gitignore instead.